src/test/java/seedu/address/logic/commands/UnregisterCommandTest.java

```diff
@@ -0,0 +1,53 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import static seedu.address.commons.core.Messages.MESSAGE_NO_SUCH_BORROWER_ID;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandFailure;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.testutil.TypicalBorrowers.ID_FIRST_BORROWER;
+import static seedu.address.testutil.TypicalBorrowers.getTypicalBorrowerRecords;
+import static seedu.address.testutil.TypicalBorrowers.getTypicalBorrowers;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.Catalog;
+import seedu.address.model.LoanRecords;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.borrower.BorrowerId;
+
+public class UnregisterCommandTest {
+
+    private Model model = new ModelManager(
+            new Catalog(), new LoanRecords(), getTypicalBorrowerRecords(), new UserPrefs());
+
+    @Test
+    public void constructor_nullBorrowerId_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> new UnregisterCommand(null));
+    }
+
+    @Test
+    public void execute_validSerialNumber_success() {
+        UnregisterCommand command = new UnregisterCommand(ID_FIRST_BORROWER);
+
+        String expectedMessage = String.format(UnregisterCommand.MESSAGE_SUCCESS, getTypicalBorrowers().get(0));
```

Comment:
instead of `getTypicalBorrowers().get(0)`, maybe you can just import `ALICE `from `TypicalBorrowers.java`